Makefile.am

```diff
@@ -233,12 +239,12 @@ tools_hogl_recover_SOURCES  = \
 		tools/recovery-engine.hpp tools/recovery-engine.cc \
 		tools/recover.cc
 tools_hogl_recover_CPPFLAGS = -I include
-tools_hogl_recover_LDADD    = src/libhogl.la $(HOGL_LIBS) $(LIBS)
+tools_hogl_recover_LDADD    = src/libhogl.la -lboost_regex $(HOGL_LIBS) $(LIBS)
```

Comment:
I don't think we need boost_regest. If c++11 is enabled we're using std::regex, otherwise it's is boost::xpresive which is header only. Must've been included in this MR by mistake.